### PR TITLE
refactor: Move Simulations to functional components

### PR DESCRIPTION
**Pull request by pjaspers**
This transforms the simulation container into a functional component (so that it's more in line with the others and that I can use the same route logic as in the sets)

It also transforms the simulation overview to be a functional component because I was getting warnings in react about using deprecated functions.

### CODE REVIEW

File: src/containers/SimulationContainer/index.js [hunk 1]
```diff
@@ -55,89 +71,70 @@ class SimulationContainer extends Component {
...
-        />
-        {isSuccess && (
-          <SimulationList key="list" simulations={jsonPayload.invoices} />
-        )}
-      </Dialog>
-    );
-  }
-}
+  return (
+    <Dialog
```
Comment (borisrorsvort) [inline src/containers/SimulationContainer/index.js]: This should move to `/components/Simulation/index.js` to keep containers html free.

**Comment by pjaspers** [comment]
@borisrorsvort this makes the container be without layout.

File: src/components/Simulation/list.js [hunk 2]
```diff
@@ -39,118 +38,100 @@ const mapOrgunits = invoices => {
...
-    const orgunits = mapOrgunits(props.simulations);
-    this.state = {
-      periods,
-      allPeriods: periods,
-      packages,
-      allPackages: packages,
-      orgUnits: [orgunits[0]],
-      allOrgUnits: orgunits,
-    };
+const SimulationList = props => {
```
Comment (borisrorsvort) [inline src/components/Simulation/list.js]: This name is confusing as these are sets tables.

File: src/components/Simulation/list.js [hunk 3]
```diff
@@ -39,118 +38,100 @@ const mapOrgunits = invoices => {
...
+  const [orgUnits, setOrgUnits] = useState([]);
+
+  // Set default selection
+  if (orgUnits.length < 1 && allOrgUnits.length > 0) {
+    setOrgUnits([allOrgUnits[0]]);
   }
+  if (periods.length < 1 && allPeriods.length > 0) setPeriods(allPeriods);
+  if (packages.length < 1 && allPackages.length > 0) setPackages(allPackages);
+
+  const filteredSimulations = simulations.filter(simulation => {
```
Comment (borisrorsvort) [inline src/components/Simulation/list.js]: Technically should happen in the SimulationContainer. But you maybe wanted to do that in a subsequent PR

Comment (borisrorsvort) [reply src/components/Simulation/list.js]: If this file is the container, it shall be moved and striped from html ^^

Comment (pjaspers) [reply src/components/Simulation/list.js]: Yes, want to fix that in an upcoming PR (which also puts the filters on the right side in the sidesheet)

Comment (pjaspers) [reply src/components/Simulation/list.js]: But this does raise the question, is there a naming convention for loaders vs managers of data? Like the `SimulationContainer` as it is in this PR is only there to fetch the data. The new container would only have the job of managing selections/visibility of the Simulation.

**Pull request merged**